docs/js-delivery-test-object.md

```diff
@@ -0,0 +1,114 @@
+    if (document.location.pathname === "/") {
+        // Activate the test at DOM Content Loaded
+        Mojito.utils.domReady(testObject.activate);
+    }
+}
+```
+
+<!--YAML 1.js-->
+```js
+function(testObject) {
+    alert("You're in an experiment...");
+}
+```
+
+<!--YAML 1.css-->
+```css
+body {
+    display:none;
+}
+```
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+
+## Test object: Root level
+
+Parameter | Description
+--|--
+**css** <br> Type: *string* <br> *Optional* | Shared CSS for the test object that is applied to all recipes. <br>`JS format`: Expects a JS string of CSS styles. <br>`YAML format`: Expects a relative path to a CSS file.
+**divertTo** <br> Type: *string* <br> *Optional* | Enable divert mode to send all eligible traffic into a particular recipe's key.
+**id** <br> Type: *string* <br> *Required* | A canonical test object ID by which subjects' assignments are recorded against. 
+**js** <br> Type: *function* <br> *Required* | Shared JS for the test object that is applied to all recipes. <br>`JS format`: Expects a valid JavaScript function on this key. <br>`YAML format`: Expects a relative path to a JS file with that function.
+**name** <br> Type: *string* <br> *Required* | The name of the test object that's useful in tracking.
+**recipes** <br> Type: *object* <br> *Required* | An object containing the definition of available recipes.
+**sampleRate** <br> Type: *number* <br> *Required* | The percentage of traffic to assign into the experiment between 0 and 1 (where 1 is 100%)
+**state** <br> Type: *string* <br> *Required* | A test object's state. Can be either: <br> `live` - in the container, accepting traffic into the experiment <br>`staging` - built into the container but not accepting traffic without a preview URL <br>`inactive` - not parsed or built into the container
+**trigger** <br> Type: *function* <br> *Required* | A JavaScript function executed as soon as the test object is loaded and can be used to conditionally activate an experiment. <br>`JS format`: Expects a valid JavaScript function on this key. <br>`YAML format`: Expects a relative path to a JS file with that function.
+
+## Recipes object
+
+Parameter | Description
+--|--
+**{{recipeId}}** <br> Type: *string* <br> *Required* | The canonical key that references a test object within the cookies and preview URLs. E.g. A recipe ID of `a` would be accesible through a preview URL like: `https://www.example.com/?mojito_w12=a`
+
+## Recipe objects sub-level
+
+Parameter | Description
+--|--
+**css** <br> Type: *string* <br> *Optional* | CSS that is applied as soon as the subject is bucketed. <br>`JS format`: Expects a JS string of CSS styles. <br>`YAML format`: Expects a relative path to a CSS file.
+**js** <br> Type: *function* <br> *Optional* | A JavaScript function that runs as soon as a subject is assigned into the test. <br>`JS format`: Expects a valid JavaScript function on this key. <br>`YAML format`: Expects a relative path to a JS file with that function.
```

Comment:
The description here should match the experiment shared js, i think?

Reply:
@allmywant - can recipe JS keys accept ```function``` and ```object``` types, too?

Reply:
Recipe JS keys accept function only.